### PR TITLE
TL improvement and Japanese Typo fix

### PR DESCRIPTION
**Pull request by msksk**
line 2027 in 015
Japanese response to negative questions has the opposite meaning of English one. It's confusing.

### CODE REVIEW

File: Update/omot_016.txt [hunk 1]
```diff
@@ -957,7 +957,7 @@ void main()
 			NULL, "\"Right.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600627", 256, TRUE);
 	OutputLine(NULL, "鷹野さんの理論は突拍子もない話ばかりですが、案外この連続事件の裏にあるもの、もし偶然じゃなかったら一筋縄ではいきそうにないですよ」",
-			NULL, " Takano-san's theories are just crazy stories, but surprisingly enough, if this series of events isn't a coincidence, they do seem to point toward that.\"", Line_Normal);
+			NULL, " Takano-san's theories are just crazy stories, but if this series of events isn't just a coincidence, we'll have a harder time taking the wraps off than we're thinking now.\"", Line_Normal);
```
Comment (msksk) [inline Update/omot_016.txt:960]: Allow me to rephrase this.
"Takano-san's theories are just crazy stories, but if this series of events isn't just a coincidence, we'll have an unexpectedly hard time taking the wraps off."

File: Update/omot_015.txt [hunk 2]
```diff
@@ -234,13 +234,13 @@ void main()
...
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "せめて連中には私を罵倒した報いを受けさせてやろうと、直接総支配人である葛西に強盗計画のことをタレ込んでやった。",
-			NULL, "Someone must've snitched the robbery plan to the general manager, Kasai. Probably hoping to get a reward by throwing me under the bus.", Line_Normal);
+			NULL, "I wanted at least to pay them back for throwing me under the bus, so I snitched the robbery plan to the general manager, Kasai.", Line_Normal);
 	ClearMessage();
 
 //rもちろん、自分のことは伏せておいたのは言うまでもない。…しかし、葛西は薄く笑った後、やおら取り出した拳銃を向けながらこう言い放ったのだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "もちろん、自分のことは伏せておいたのは言うまでもない。…しかし、葛西は薄く笑った後、やおら取り出した拳銃を向けながらこう言い放ったのだ。",
-			NULL, "Of course, it goes without saying that I did my part to keep it secret. ...But Kasai lightly laughed at us and then spoke while pulling out a pistol.", Line_Normal);
```
Comment (msksk) [inline Update/omot_015.txt:243]: Rephrase No.2
”Of course, it goes without saying that I kept my participation secret from him.”

File: Update/omot_016.txt [hunk 3]
```diff
@@ -5014,7 +5014,7 @@ void main()
 //r詩音の声が聞こえ、そばでうずくまる沙都子に駆けてきて抱きしめる気配がする。が、私はそれよりも今さっき、耳に残った声を確かめようと、おそるおそる顔を…上げた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "詩音の声が聞こえ、そばでうずくまる沙都子に駆けてきて抱きしめる気配がする。が、私はそれよりも今さっき、耳に残った声を確かめようと、おそるおそる顔を…上げた。",
-			NULL, "I heard Shion cry out, and then she rushed over to my side to embrace Satoko. But more importantly, I had to confirm that voice that was still lingering in my ears... so I looked up.", Line_Normal);
+			NULL, "I heard Shion cry out and sensed she rush over to my side to embrace Satoko. But more importantly, I had to confirm that voice that was still lingering in my ears... so I timidly looked up.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/omot_016.txt:5017]:
```suggestion
			NULL, "I heard Shion cry out and sensed her rushing over to my side to embrace Satoko. But more importantly, I had to confirm that voice that was still lingering in my ears... so I timidly looked up.", Line_Normal);
```

File: Update/omot_016.txt [hunk 4]
```diff
@@ -5220,7 +5220,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 13, "ps3/s14/13/211300094", 256, TRUE);
 	OutputLine(NULL, "「ああ、それは大石さんたちの仕事だから。",
-			NULL, "\"Yeah, that's Ooishi's job.", Line_WaitForInput);
+			NULL, "\"Yeah, that's Ooishi-san and his coworkers' job.", Line_WaitForInput);
```
Comment (DoctorDiablo) [inline Update/omot_016.txt:5223]:
```suggestion
			NULL, "\"Yeah, that's Ooishi's job.", Line_WaitForInput);
```

**Review by DoctorDiablo — APPROVED** [review]

File: Update/omot_015.txt [hunk 5]
```diff
@@ -240,7 +240,7 @@ void main()
...
+			NULL, "Of course, it goes without saying that I did my part to keep it secret. ...But Kasai lightly laughed at me and then spoke while pulling out a pistol.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/omot_015.txt:243]:
```suggestion
			NULL, "Of course, it goes without saying that I kept my involvement secret from him. ...But Kasai lightly laughed at me and then spoke while pulling out a pistol.", Line_Normal);
```

**Review by DoctorDiablo — APPROVED** [review]

**Pull request merged**